src/test/java/SetTest.java

```diff
@@ -0,0 +1,55 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach // test를 하기 전 무조건 세팅되는 값
+    void setUp() {
+        numbers = new HashSet<>();
+
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size() 메서드를 사용해 Set의 크기를 확인한다.")
+    void confirmSize() {
+        Assertions.assertThat(numbers.size()).isEqualTo(3);
```

Comment:
`static import`를 활용하면 `assertThat()` 검증 시 매번 `Assertions.`를 추가하지 않아도 될 것 같습니다.
```suggestion
        assertThat(numbers.size()).isEqualTo(3);
```

Reply:
감사합니다!